modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
Discussion seems to be gradually going off scope of items in this ticket :)  

I recommend that we create a ticket for fixing behaviour of red loader, and a ticket to create loading state indicators for the FileTable. iirc Alex and Shane wanted a skeleton state, maybe something like ICGC's projects table

![image](https://cloud.githubusercontent.com/assets/743976/24463519/f211f670-1474-11e7-9314-6c7262c7941c.png)